common/common.go

```diff
@@ -629,3 +647,87 @@ func CheckDir(dir string, create bool) error {
 	}
 	return nil
 }
+
+// PromptForUsername prompts for username stored in database and returns a
+// string from stdin
+func PromptForUsername() (string, error) {
+	log.Info("Please enter username to connect to the database:")
+	var username string
+	for {
+		i, err := fmt.Scanln(&username)
```

Comment:
Scanln usage here will error if a user enters a blank username (as its considered a new line \n) or a username that enters with a space if not properly stripped

Reply:
What do you suggest?